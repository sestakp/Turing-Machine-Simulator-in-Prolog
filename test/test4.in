S a S L
aaaaaa