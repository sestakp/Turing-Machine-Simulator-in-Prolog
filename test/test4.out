The Turing machine stopped abnormally. Access to position -1
